viewer/packages/api/src/public/migration/types.ts

```diff
@@ -79,6 +79,13 @@ export interface Cognite3DViewerOptions {
    */
   rendererResolutionThreshold?: number;
 
+  /**
+   * A factor that will scale down the resolution when moving the camera. This can
+   * be used to achieve a better user experience on devices with limited hardware.
+   * A value of e.g. 0.25 will approximately divide the number of pixels on the screen by four.
+   */
+  movingResolutionFactor?: number;
```

Comment:
As above, I suggest to make this `cameraMovingResolutionFactor` or something. Enforce values between 0 and 1.